proposals/0266-New-vehicle-data-GearStatus.md

```diff
@@ -173,8 +173,8 @@ We need to add `GearStatus` for `GetVehicleData`, `SubscribeVehicleData`, `Unsub
 
 ```xml	
 <struct name="GearStatus">
-	<param name="selectedGear" type="Common.PRNDL" mandatory="false">
-		<description>Tells the gear drive mode i.e. PARK, DRIVE, SPORT etc</description>
+	<param name="userSelectedGear" type="Common.PRNDL" mandatory="false">
+		<description>Gear position selected by the user i.e. Park, Drive, Reverse</description>
 	</param>
 	<param name="actualGear" type="Common.PRNDL" mandatory="false">
 		<description>Tells the actual gear selected by transmission</description>
```

Comment:
Based on https://github.com/smartdevicelink/sdl_evolution/issues/881#issuecomment-573838656, this should be
```suggestion
		<description>Actual Gear in use by the transmission</description>
```

Reply:
Done